### DISCUSSION TITLE
阿里云盘分享中的视频无法播放

### DISCUSSION THREAD

**Discussion by ManjuSage** [AlistGo/alist#3505] [Q&A]
阿里云盘分享的视频无法播放，其他云盘播放的视频都是正常的。
![屏幕截图 2023-02-18 182121](https://user-images.githubusercontent.com/84658164/219855580-c3078275-10b0-4d6f-985f-879a1976f240.png)
用potplayer打开阿里云盘分享也不行，打开其他都可以。
![屏幕截图 2023-02-18 183652](https://user-images.githubusercontent.com/84658164/219855828-222dccaa-9633-481c-a7d2-0fb122e345f4.png)
而且阿里云盘分享的内容转存到自己的阿里云盘中alist可以播放。

**Comment by xxxzsgxxx** [comment]
我的问题和你不太一样，最新版的alist重装后，目前可以在页面播放视频，但无法下载。
